url/urltestdata.json

```diff
@@ -4587,5 +4587,76 @@
     "pathname": "/",
     "search": "",
     "hash": ""
+  },
+  "# More file URL tests by zcorpan",
+  {
+    "input": "/",
+    "base": "file:///C:/a/b",
+    "href": "file:///C:/",
+    "protocol": "file:",
+    "username": "",
+    "password": "",
+    "host": "",
+    "hostname": "",
+    "port": "",
+    "pathname": "/",
+    "search": "",
+    "hash": ""
+  },
+  {
+    "input": "//d:",
+    "base": "file:///C:/a/b",
+    "href": "file:///d:",
```

Comment:
That seems correct, but then pathname wouldn't end in a "/" either. It's also not entirely clear to me that is the desired outcome.